### ISSUE TITLE
Update "Login error not announced by screen reader - is this a fail?"

### ISSUE THREAD

**Issue by Jefferydo** [cenw-wscoe/WCAG-in-Plain-Language#56]
The solution only mentions aria-live notifications but there are several more techniques. Also, if the login button triggers a page refresh, aria-live is not the ideal technique. It's best-suited to inline feedback when the user move focus away from the problem control.

Course page on [User notifications](https://bati-itao.github.io/learning/esdc-self-paced-web-accessibility-course/module6/user-notifications.html). Our section on in-line feedback doesn't mention or demonstrate aria-live, so need to update the page. This is the gist of the advice:

As per the examples below, provide feedback:

- in the main heading (optional)
- in the <title> element (optional)
- as a list of errors before the form
- In-line, programmatically associated with the form control
